### DISCUSSION TITLE
authentification

### DISCUSSION THREAD

**Discussion by MasterThreadInf** [unoplatform/uno#10440] [Q&A]
I'm trying to make an sample soultion for uno authentification and I followed steps as per this link https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Tutorials/Authentication/HowTo-Authentication.html 
and created all the things needed. I also added extension template https://platform.uno/docs/articles/external/uno.extensions/doc/Overview/ExtensionsOverviewAndGettingStarted.html 

Unfortunately, I'm getting same Error	
CS0006	Metadata file 'C:\Users\source\repos\UnoAuthentification\authen\authen\obj\Debug\net6.0\ref\authen.dll' could not be found	authen.Wasm	C:\Users\source\repos\UnoAuthentification\authen\authen.Wasm\CSC

and .dll is really missing but what is it I'm doing wrong?
I've also added workloads needed for each platform.

**Comment by nickrandolph** [comment]
Hey before we try working out the issue with auth, was your sample building and running ok before adding the Uno extensions?

**Reply by MasterThreadInf** [reply]
Hi ,ya before extensions it was building and running.

**Reply by MasterThreadInf** [reply]
Hi ,
I tried again the same thing and got little further , now I'm getting error which says 
UXAML0001 : Unable to find member ViewModel on type MyProject.Views.MainPage 

after adding following code in MainPage.Xaml
`<Button 
            Click="{x:Bind ViewModel.Authenticate}" 
            Content="Login" />`
Is there anything I need to add in MainPage.Xaml ?
